src/user/settings.js

```diff
@@ -74,8 +74,7 @@ module.exports = function (User) {
 				settings.categoryTopicSort = getSetting(settings, 'categoryTopicSort', 'newest_to_oldest');
 				settings.followTopicsOnCreate = parseInt(getSetting(settings, 'followTopicsOnCreate', 1), 10) === 1;
 				settings.followTopicsOnReply = parseInt(getSetting(settings, 'followTopicsOnReply', 0), 10) === 1;
-				settings.sendChatNotifications = parseInt(getSetting(settings, 'sendChatNotifications', 0), 10) === 1;
-				settings.sendPostNotifications = parseInt(getSetting(settings, 'sendPostNotifications', 0), 10) === 1;
+				settings.upvoteNotifFreq = getSetting(settings, 'upvoteNotifFreq', 'all');
```

Comment:
:point_left: aren't these settings needed?

Reply:
No longer needed after my changes to notification delivery settings.